pages/convince-your-boss.tsx

```diff
@@ -0,0 +1,170 @@
+import Link from "next/link";
+import CenteredButton from "../ui/centered-button";
+import Divider from "../ui/divider";
+import ColoredSection from "../ui/colored-section";
+import SocialShare from "../ui/social-share";
+
+function CallForPapers() {
+  return (
+    <>
+      <SocialShare
+        title="Convince your boss!"
+        description=""
+        socialShareImage="social-share/convince-your-boss.png"
+      />
+      <Divider>
+        <h1>Convincing Your Employer to Invest in Your Development</h1>
```

Comment:
This is a long title and does not look good on most of the resolutions

Works on 1700+px and 1240-1280px

Reply:
fixed!